### DISCUSSION TITLE
Code Owners review for specifc branches

### DISCUSSION THREAD

**Discussion by LachlanNewman** [community/community#39613] [Enterprise]
### Select Topic Area

Question

### Body

Is there a way to require reviews from specific teams for PRs to certain branches 
for example:

`feat/1 -> develop ` requires approval from `@org/team-develop`
`develop -> main`   requires approval from `@org/team-main`

**Comment by WWooodyy** [comment]
Your best bet would be checking the Branch protection rules tab,
However I believe that this may be possible with Github [Enterprise](https://github.com/enterprise).

I've worked at an organisation before which has had this type of setup.

**Reply by LachlanNewman** [reply]
Do you remember how they set it up ?
